docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -13,18 +13,22 @@ Synopsis
 Description
 -----------
 
-Returns approximated statistics for the named table or for the results of a query.
+Returns approximated statistics for the named table or for the results of a
+query. Returns ``NULL`` for any statistics that are not populated or
+unavailable on the data source.
 
-Statistics are returned for each column, along with a summary row.
+Statistics are returned for each column, along with a summary row. The following
+table lists the statistics that Trino supports. Any additional statistics
+collected on the data source, other than those listed here, are not returned.
 
 ==========================  =============================================================
 Column                      Description
 ==========================  =============================================================
 ``column_name``             The name of the column (``NULL`` for the summary row)
```

Comment:
is this one table to cover stats for all tables, or is there a separate stats table for each table?

Reply:
I would think that the summary row would be for a table (hence the ``NULL`` value for column name in summary row, because most of the stats listed don't describe tables.

Reply:
If I'm understanding correctly, it will return one summary row for each table, in addition to a row that describes each column in that table. I reworded the intro paragraph to make that more clear.